ee/test/test_migration_0012.py

```diff
@@ -0,0 +1,103 @@
+from django.db import transaction
+from django.db.models import Q
+
+from posthog.models import Tag as TagModel
+from posthog.models import TaggedItem as TaggedItemModel
+from posthog.models import Team
+from posthog.test.base import TestMigrations
+
+
+class TagsTestCase(TestMigrations):
```

Comment:
If you're adding tests I suggest using the snapshot functions here as well to document the generated SQL